config/routes.js

```diff
@@ -329,6 +329,11 @@ module.exports.routes = {
     action: 'listDatastreams',
     csrf: false
   },
+  'get /:branding/:portal/api/records/datastreams/:oid': {
+    controller: 'webservice/RecordController',
+    action: 'listDatastreams',
+    csrf: false
+  },
```

Comment:
@andrewbrazzatti Did you intend to add another route for `listDatastreams`, or change the existing route from `put` to `get`?